src/storage/src/render/upsert.rs

```diff
@@ -156,7 +156,7 @@ pub fn rehydration_finished<G, T>(
     let worker_id = source_config.worker_id;
     let id = source_config.id;
     let mut builder = AsyncOperatorBuilder::new(format!("rehydration_finished({id}"), scope);
-    let mut input = builder.new_input(input, Pipeline);
+    let mut input = builder.new_disconnected_input(input, Pipeline);
```

Comment:
why can this be disconnected?

Reply:
What other option is there? There are no outputs :)